src/extension.ts

```diff
@@ -1,51 +1,81 @@
 import * as vscode from 'vscode';
-import { TabInputCustom, TabInputNotebook, TabInputNotebookDiff, TabInputTerminal, TabInputText, TabInputTextDiff, TabInputWebview, Uri } from 'vscode';
+import {
+  TabInputCustom,
+  TabInputNotebook,
+  TabInputNotebookDiff,
+  TabInputTerminal,
+  TabInputText,
+  TabInputTextDiff,
+  TabInputWebview,
+  Uri,
+} from 'vscode';
 
-type TabContentData = TabInputText | TabInputTextDiff | TabInputCustom | TabInputWebview | TabInputNotebook | TabInputNotebookDiff | TabInputTerminal | unknown;
+type TabContentData =
+  | TabInputText
+  | TabInputTextDiff
+  | TabInputCustom
+  | TabInputWebview
+  | TabInputNotebook
+  | TabInputNotebookDiff
+  | TabInputTerminal
+  | unknown;
+
+let changeTabs = (withNumber: boolean) => async () => {
```

Comment:
Not a big typescript user, is this `let name = ...` the preferred way to define a function instead of `function changeTabs(withNumber: boolean): () => Promise<...>`?

Regardless, I do think `const` makes more sense here than `let`. I would also prefer all these functions to be consistent whether it's using `function` or not.

I know i used `let` below when creating the disposable and the code block you pulled this code from, just looking to tighten up this kind of thing while we're making changes like this.